ubuntudesign/django-json-redirects
# Use a YAML file instead of a JSON one

This actually requires minimal changes for any project.

All you have to do is rename `redirects.json` to `redirects.yaml` and
change the module from `django_json_redirect` to `django_yaml_redirects`.

I've also taken the liberty of fixing the encoding issue with UTF-8 characters.

QA
---

``` bash
git clone git@github.com:ubuntudesign/www.ubuntu.com
cd www.ubuntu.com
mv redirects.json redirects.yaml
vim requirements/standard.txt  # Add django_yaml_redirects version 0.2
vim webapp/urls.py  # Change `django_json_redirects` to `django_yaml_redirects`
make clean-images run
```

Visit the site, check it works just as well as it did before.
